app/core/data/evaluate_rag.py

```
@@ -0,0 +1,575 @@
+                 "generate_questions_prompt"):
+        prompt_obj = getattr(ar_metric, attr, None)
+        if prompt_obj is None:
+            continue
+        # StringPrompt / PromptTemplate / PydanticPrompt 계열 (instruction 속성)
+        if hasattr(prompt_obj, "instruction"):
+            try:
+                prompt_obj.instruction = korean_prefix + (prompt_obj.instruction or "")
+                patched = True
+                break
+            except Exception:
+                pass
```

Ruff (0.15.11)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)